src/main/java/org/carlmontrobotics/subsystems/Limelight.java

```diff
@@ -0,0 +1,72 @@
+package org.carlmontrobotics.subsystems;
+
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+
+public class Limelight extends SubsystemBase {
+  //public boolean targetValid = false;
+  //private double tid, tv, tx, ty, ta;
+  //private double distOffset, horizOffset;
+  //private double horizHeadingError, horizAdjust;
+
+  public Limelight() {}
+
+  @Override
+  public void periodic() {
+    //tracking();
+  }
+
+  public void tracking() {
+    /* gets certain limelight values (ta, tx, ty, tv, tid, etc)
+    checks if there is a target and updates targetValid
+    puts those values to SmartDashboard
+    */
+    
+    /* 
+     * tid = NetworkTableInstance.getDefault().getTable(“limelight”).getEntry(“tid”).getDouble(0);
+     * tv = NetworkTableInstance.getDefault().getTable(“limelight”).getEntry("tv").getDouble(0);
+     * tx = NetworkTableInstance.getDefault().getTable(“limelight”).getEntry("tx").getDouble(0);
+     * ty = NetworkTableInstance.getDefault().getTable(“limelight”).getEntry("ty").getDouble(0);
+     * ta = NetworkTableInstance.getDefault().getTable(“limelight”).getEntry("ta").getDouble(0);
+     * 
+     * if (tv == 1.0){
```

Comment:
its dangerous to use == for double since if the value is a tiny bit off even if its really close the condition will not be true.

Reply:
also use `targetValid = (tv == 1.0)` instead of the if statements (again, don't use the `==`)

Reply:
i used == because tv can only be 0.0 or 1.0 according to the documentation